### DISCUSSION TITLE
Multiple accounts / logins for GitHub Codespaces VSCode Extension

### DISCUSSION THREAD

**Discussion by trevorstr** [community/community#78158] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello, I use VSCode locally, from multiple different systems, with the GitHub Codespaces extension, to access remote Codespaces.

I have at least two separate GitHub accounts, one personally, one for work, and others for specific client projects.

When I log into GitHub Codespaces, I only have the option of authenticating to a single account. It doesn't seem possible to login to the Codespaces extension with multiple accounts. 

Will this feature be available someday? Thank you

**Comment by KnisterPeter** [comment]
I have the same issue and would be interested in any update on this.
